indico/modules/networks/controllers.py

```diff
@@ -0,0 +1,96 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from flask import request, session
+from sqlalchemy.orm import contains_eager
+
+from MaKaC.webinterface.rh.admins import RHAdminBase
+from indico.core.db import db
+from indico.modules.categories.models.categories import Category
+from indico.modules.events.models.events import Event
+from indico.modules.networks import logger
+from indico.modules.networks.forms import IPNetworkGroupForm
+from indico.modules.networks.models.networks import IPNetworkGroup
+from indico.modules.networks.views import WPNetworksAdmin
+from indico.web.util import jsonify_data, jsonify_form, jsonify_template
+
+
+class RHManageNetworks(RHAdminBase):
+    """Management list for IPNetworks"""
+
+    def _process(self):
+        network_groups = IPNetworkGroup.find().order_by(IPNetworkGroup.name).all()
+        return WPNetworksAdmin.render_template('networks.html', network_groups=network_groups)
+
+
+class RHCreateIPNetworkGroup(RHAdminBase):
+    """Dialog to create an IPNetworkGroup"""
+
+    CSRF_ENABLED = True
```

Comment:
I'd add a base class used by all other RHs in here which already enables CSRF (we want it globally enabled in 2.0 and since we need to manually check RHs not setting it it's better to set it everywhere, even on pages that technically don't need it)